src/gettsim/tests_germany/policy_cases/full_taxes_and_transfers/2019-01-01/hh_id_2.yaml

```diff
@@ -123,6 +123,11 @@ inputs:
           - 0.0
           - 0.0
           - 0.0
+        aus_berufsständischen_versicherungen_m:
```

Comment:
Just a random occurrence: I am surprised we do not need `alter_beginn_leistungsbezug_sonstige_private_vorsorge` among the inputs. Why don't we need it in all these tests but do require it in the (current version of) the Elterngeld example notebook?